test/api/core/entrypoints/protocols/websocket.test.js

```diff
@@ -651,6 +651,50 @@ describe('/lib/api/core/entrypoints/protocols/websocket', () => {
       should(activeConnection.socket.terminate).not.be.called();
       should(activeConnection.socket.ping).not.be.called();
     });
+
+    it('should mark a socket as dead if not available for PING', () => {
```

Comment:
Comes from https://github.com/kuzzleio/kuzzle/pull/1408. Do we need to depend on it?